sky/serve/examples/llama2/llama2.yaml

```diff
@@ -21,10 +21,7 @@ setup: |
   fi
 
   # Install dependencies
-  pip install git+https://github.com/lm-sys/FastChat.git
-  # Need the latest transformers to support 70B model
-  pip install git+https://github.com/huggingface/transformers.git
-
+  pip install "fschat[model_worker,webui]==0.2.24"
```

Comment:
This example is borrowed from the master branch. Maybe we should fix the master too?

Reply:
yes we should..!